app/js/dash/TimelineConverter.js

```diff
@@ -50,8 +50,9 @@ Dash.dependencies.TimelineConverter = function () {
             return calcAvailabilityTimeFromPresentationTime.call(this, presentationTime, mpd, isDynamic, true);
         },
 
-        calcPresentationTimeFromWallTime = function (wallTime, period) {
-            return ((wallTime.getTime() - period.mpd.availabilityStartTime.getTime() + clientServerTimeShift * 1000) / 1000);
+        calcPresentationTimeFromWallTime = function (wallTime, period, ignoreOffset) {
+            var offset = (ignoreOffset ? 0 : clientServerTimeShift) * 1000;
```

Comment:
I do not like ignoreOffset. the output of time sync or live edge search should be to set the offset so that the offset is never ignored.